README.md

```diff
@@ -12,49 +12,31 @@ To run the pipeline, you need access to an FTP server.
 If you or your organization is interested in using the pipeline, 
 please contact [510 Global](https://www.510.global/contact-us/)
 to obtain the credentials.  You will receive a file called `settings.py`, which you need to place in 
-the `lib` directory.
+the `IBF-Typhoon-model/lib` directory.
 
 ### Without Docker
 
-The main script for the pipeline is IBF-Typhoon-model/mainpipeline.py.
+The main script for the pipeline is `IBF-Typhoon-model/scr/pipeline.py`.
 It can in principle be run locally by setting up your local Python environment using
-the `IBF-Typhoon-model/requirements.txt` file, and installing all the R packages listed in the `IBF-Typhoon-model/Dockerfile`.
+the `requirements.txt` file, and installing all the R packages listed in the `Dockerfile`.
 However, we suggest using the docker image instead.
 
 ### With Docker
 
 You will need to have docker installed, and you should modify `Docker-settings`
 to set the container memory to at least 2GB
 
-####  Get the image
+####  Build the image
 
-The docker image is available at 
-[typhoonibf:latest](https://hub.docker.com/repository/docker/rodekruis510/typhoonibf/).
-To download it, run:
-```
-docker pull rodekruis510/typhoonibf
-```
-
-You can also build it yourself. To do so, first download 
-[this version of ncdf4](https://cran.r-project.org/src/contrib/Archive/ncdf4/ncdf4_1.13.tar.gz)
-and place the tarball in the top-level directory.
-Then run:
+To build the docker image, run:
 ```
 docker build -t rodekruis510/typhoonibf .
 ```
-Create a docker container
-```
-docker run --name typhoonibf rodekruis510/typhoonibf
-
-```
-Run and access the container
-```
-docker run -it --entrypoint /bin/bash rodekruis510/typhoonibf
-```
 ## Running
-To spin up and enter the docker container, execute:
+To spin up and enter the docker container, create a directory where you would like
+the output data to go. Then run:
```

Comment:
what about simplifying it by removing the need to insert path to output-directory? i have the following in mind based on your feedback from yesterday 
change directory to "IBF-Typhoon-model" and execute:
docker run --rm -it --name=fbf-phv3 -v IBF-Typhoon-model:/home/fbf  rodekruis510/typhoonibf bash

Reply:
The command that you're proposing actually does something slightly different, which I will explain. The way I see it, we have two possible options for building the image & container: 

1. Copy the code into the image (`ADD IBF-Typhoon-model .`), and then mount only the output directory ("forecast"). This is what I've implemented here: I've left the code-copying command in the dockerfile. The reason that you still have to mount the output directory (that's what the  `-v $path-to-output-directory:/home/fbf` portion of the command does) is that otherwise the docker container will download and save all files _in the container_, and they will not be accessible outside of the container (and will disappear once the container is removed). 

Advantages: probably the "right thing to do" (i.e. having the code in the image)
Disadvantages: Can't share the image for now because it contains the settings file

2. Don't copy the code into the image (take away the `ADD IBF-Typhoon-model .` command) and mount the full `IBF-Typhoon-model` directory, as you have done above. 

Advantages: Image is a bit lighter, can edit code locally and changes will show up in the container
Disadvantages: Image isn't really "complete" (doesn't contain code to run), room for error if local edits aren't committed

We need to choose between these options. Right now option 1 is implemented but I don't think I feel very strongly so it's up to you.